src/cmd-buildextend-metal

```diff
@@ -72,6 +83,17 @@ if [[ "$basearch" != "s390x" && $image_type == dasd ]]; then
     fatal "$basearch is not supported for building dasd images"
 fi
 
+# SecureExecution extra stuff
+secex_image_suffix=
+secex_extra_opts=
+secex_hostkey_drive=()
+if [[ $secure_execution -eq "1" ]]; then
+    secex_image_suffix="-secex"
+    secex_extra_opts="--with-secure-execution"
```

Comment:
This can just add to `disk_args` instead. We'd have to move up its declaration or alternatively move this down.